AAP-37117 adding a proc on decoding the base64 password (#2675)
ansible/aap-docs: downstream/modules/platform/proc-cli-get-controller-pwd.adoc

[id="proc-cli-get-controller-pwd{context}"]

= Fetching {Gateway} login details from the {OCPShort} CLI

To login to the {Gateway}, you need the web address and the password.

> == Fetching the {Gateway}y web address

Reviewer: ```suggestion
== Fetching the {Gateway} web address
```

example   example-ansible-automation-platform.apps-crc.testing          example-service   http   edge/Redirect   None
-----

The address for the {Gateway} instance is `example-ansible-automation-platform.apps-crc.testing`.

== Fetching the {Gateway} password

The YAML block for the {Gateway} instance in the `AnsibleAutomationPlatform` object assigns values to the _name_ and _admin_user_ keys.

. Use these values in the following command to fetch the password for the {Gateway} instance.

-----
oc get secret/<your instance name>-<admin_user>-password -o yaml
-----

> . The default value for _admin_user_ is `_admin_`. Modify the command if you changed the admin username in the `AnsibleAutomationPlatform` object.

Reviewer: This is showing as step 1 in the built doc:
![image](https://github.com/user-attachments/assets/bd02957e-94b2-4a1f-bd7c-6f68e2eea86a)

  name: example-admin-password
  namespace: ansible-automation-platform

----- 

== Decoding the {Gateway} password

> After you have found your gateway password, you will need to decode it from base64. 

Reviewer: ```suggestion
After you have found your gateway password, you must decode it from base64. 
```
